scanner/src/main/kotlin/ScanController.kt

```diff
@@ -65,10 +65,10 @@ internal class ScanController(
     private val nestedProvenanceResolutionIssues = mutableMapOf<Identifier, Issue>()
 
     /**
-     * A map of [Identifier]s associated with a list of [Issue]s that occurred during a scan besides the issues
+     * A map of [Identifier]s associated with a set of [Issue]s that occurred during a scan besides the issues
```

Comment:
I'm not sure if this should already close #10054 because ORT does still provide no way to delete broken scan results from the storage that were created before this change. @sschuberth What do you think?

Reply:
You have a point there, but I guess the best we could do is to provide users with a tool / helper-cli command to remove those bogus stored scan results with issues that contain a certain kind of message. As we'd never know if and when users would run the tool, I'd also be fine with closing the issue now, as the root cause is fixed, and provide the mentioned tooling as a follow-up.

Reply:
I'll leave the decision to you two, I'm fine with either outcome.

Reply:
I created https://github.com/oss-review-toolkit/ort/pull/10495 to address that part.